runelite-client/src/test/java/net/runelite/client/plugins/screenshot/ScreenshotPluginTest.java

```diff
@@ -422,6 +434,29 @@ public void testCollectionLogChat()
 		when(client.getVar(Varbits.COLLECTION_LOG_NOTIFICATION)).thenReturn(3);
 		screenshotPlugin.onChatMessage(chatMessageEvent);
 
+		verify(drawManager).requestNextFrameListener(any(Consumer.class));
+	}
+
+	@Test
+	public void testCombatAchievementsPopup()
+	{
+		ScriptPreFired notificationStart = new ScriptPreFired(ScriptID.NOTIFICATION_START);
+		screenshotPlugin.onScriptPreFired(notificationStart);
+
+		when(client.getVar(VarClientStr.NOTIFICATION_TOP_TEXT)).thenReturn("Combat Task Completed!");
+		when(client.getVar(VarClientStr.NOTIFICATION_BOTTOM_TEXT)).thenReturn("Task Completed: <col=ffffff>Handyman</col>");
+
+		ScriptPreFired notificationDelay = new ScriptPreFired(ScriptID.NOTIFICATION_DELAY);
+		screenshotPlugin.onScriptPreFired(notificationDelay);
+	}
+
+	@Test
+	public void testCombatAchievementsChat()
+	{
+		ChatMessage chatMessageEvent = new ChatMessage(null, GAMEMESSAGE, "", COMBAT_ACHIEVEMENTS_CHAT, null, 0);
+
+		screenshotPlugin.onChatMessage(chatMessageEvent);
+
 		verifyNoMoreInteractions(drawManager);
```

Comment:
this should probably be testing that a screenshot is taken right? why is it not doing that?